### PR TITLE
Added success message and loading animation while creating or editing…

### PR DESCRIPTION
**Pull request by Scttpr**
… CRUD

Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Scttpr** [comment]
[Bouton sauvegarder : Afficher un état quand en chargement / Etat](https://trello.com/c/SlcAYU4x/161-bouton-sauvegarder-afficher-un-%C3%A9tat-quand-en-chargement-etat)

File: pages/_campus/cars/_id/edit.vue [hunk 1]
```diff
@@ -65,6 +65,8 @@
         v-if="id"
         type="submit"
         class="button is-primary"
+        :class="loading && 'is-loading'"
```
Comment (Nainterceptor) [inline pages/_campus/cars/_id/edit.vue]: I don't like this syntaxe, maybe better to the following, you don't think ? 
```js
:class="{'is-loading': loading}"
```

File: pages/_campus/cars/_id/edit.vue [hunk 2]
```diff
@@ -135,6 +140,10 @@ export default {
           this.$toast.error('Erreur serveur, si le problème persiste, veuillez contacter le service technique');
         }
       }
+      this.toggleLoading();
+    },
+    toggleLoading() {
```
Comment (Nainterceptor) [inline pages/_campus/cars/_id/edit.vue]: Can you create a mixin for that ? And, be able to force a value in toggle function (be able to do `this.toggleLoading(true)` or `this.toggleLoading(false)`), and use this parameter.

File: helpers/mixins/toggle.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/prefer-default-export
+export const toggle = {
```
Comment (Nainterceptor) [inline helpers/mixins/toggle.js]: toggle is too generic, something like your previous naming `toggleLoading` maybe better, and prevent conflicting

File: helpers/mixins/toggle.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/prefer-default-export
```
Comment (Nainterceptor) [inline helpers/mixins/toggle.js]: You can rename file toggle-loading too, and use default export.

**Pull request merged**